package.json

```diff
@@ -34,7 +34,7 @@
   },
   "homepage": "https://github.com/electricimp/imp-central-impt",
   "dependencies": {
-    "Builder": "https://github.com/deldrid1/Builder#hotfix/remote-relative-includes",
+    "Builder": "https://github.com/EatonGMBD/Builder#develop",
```

Comment:
Should this be a dependency on `develop` or should this be a versioned released (even if it's marked as a pre-release?)

Reply:
It is just a temporary dependency. Once Builder is released, we will change the dependency here to a tagged released version.